src/client/components/homeComponents/HomeRatings.jsx

```diff
@@ -1,14 +1,118 @@
-import pikachuImg from "../../assets/pikachuTest.png";
+import { useState, useEffect } from 'react';
+
+const URL = process.env.REACT_APP_API_URL;
+const pokemonRatingsEndpoint = "/pokemon/ratings";
+const pokemonRatingsURL = URL + pokemonRatingsEndpoint;
+
 
 const HomeRatings = () => {
+    const [rawData, setRawData] = useState([]);
+    const [avgRatingsArr, setAvgRatingsArr] = useState([]);
+
+    /* setting states of sorted arrays by rating type for all pokemon */
+
+    let [sortedByDesign, setSortedByDesign] = useState([]);
+    let [sortedByStrength, setSortedByStrength] = useState([]);
+    let [sortedByCompanion, setSortedByCompanion] = useState([]);
+
+    /* function to calculate the average rating by type used in returnAvgRatings function */
+
+    const calcAvg = (type, ratingArr) => {
+        let ratingSum = 0;
+        for (let i = 0; i < ratingArr.length; i++) {
+            ratingSum = ratingSum + ratingArr[i][type];
+        }
+        let ratingAvg = ratingSum / ratingArr.length;
```

Comment:
Be careful with your use of `let`'s, `ratingAvg` never gets changed after being declared so it should be a const